python/cudf/cudf/core/column/string.py

```diff
@@ -3623,6 +3623,60 @@ def findall(self, pat: str, flags: int = 0) -> SeriesOrIndex:
         data = libstrings.findall(self._column, pat, flags)
         return self._return_or_inplace(data)
 
+    def find_multiple(self, patterns: SeriesOrIndex) -> "cudf.Series":
+        """
+        Find all first occurrences of patterns in the Series/Index.
+
+        Parameters
+        ----------
+        patterns : Series of Index
+            Patterns to search for in the given Series/Index.
+
+        Returns
+        -------
+        Series
+            A Series with a list of all indices
+            if the pattern's first occurrence.
+            -1 if a pattern is not found.
+
+        Examples
+        --------
+        >>> import cudf
+        >>> s = cudf.Series(["strings", "to", "search", "in"])
+        >>> s
+        0    strings
+        1         to
+        2     search
+        3         in
+        dtype: object
+        >>> t = cudf.Series(["a", "s", "g", "i", "o", "r"])
+        >>> s.str.find_multiple(t)
+        0       [-1, 0, 5, 3, -1, 2]
+        1    [-1, -1, -1, -1, 1, -1]
+        2      [2, 0, -1, -1, -1, 3]
+        3    [-1, -1, -1, 0, -1, -1]
+        dtype: list
+        """
+        if not isinstance(patterns, (cudf.Series, cudf.Index)):
```

Comment:
So we don't want to support a list of strings, tuple of strings, or a pandas Series? We have some kind of conversion functions for host data, I would guess, but I don't know what exactly to recommend.

Reply:
Done